README.md

```diff
@@ -1,19 +1,22 @@
-# A PHP client for Slack's API
+# Introduction
```

Comment:
"Introduction" as the main Readme title is kind of ill. The previous title was better, is that a docsify issue?

Reply:
Nope, I started by including the readme directly in the documentation. But in the end I no longer use it, so I will restore this title :+1: